src/contracts/pods/EigenPodManager.sol

```diff
@@ -177,6 +176,19 @@ contract EigenPodManager is Initializable, OwnableUpgradeable, Pausable, IEigenP
         _setMaxPods(newMaxPods);
     }
 
+    function decrementWithdrawableRestakedExecutionLayerGwei(address podOwner, uint256 amountWei) 
```

Comment:
def need natspec comments on these. also not sure how we are handling communication from the StrategyManager, should this go thru the EPM. cc @ChaoticWalrus

Reply:
not sure, seems like all the communication from pods to SM is thru EPM but up to you guys

Reply:
I realize it's a bit more inefficient but I think being consistent w/ the call routing is worth it for the clarity + consistency of design.
Agreed NatSpec is needed/missing.